cms/envs/common.py

```diff
@@ -1249,3 +1249,5 @@
 
 # NodeBB settings
 NODEBB_ENDPOINT = 'http://community.philanthropyu.org'
```

Comment:
All three of these settings don't need to be defined here. You should just import them from `lms.env.common` as we are importing many of them already.